### PR TITLE
Add Websockets example

### PR DESCRIPTION
**Pull request by davorrunje**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: samples/apps/websockets/README
Comment (jackgerrits) [inline samples/apps/websockets/README:1]: can you add the md file extension

File: samples/apps/websockets/README [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Running the example
+
+1. Navigate to the directory containing the example:
+    ```
+    cd /workspaces/autogen/samples/apps/websockets
+    ```
+
+2. Install the necessary dependencies:
+    ```
+    ./setup.sh
```
[the remark this answers is not in the document]
Comment (davorrunje) [reply samples/apps/websockets/README]: It is `setup.py` now :)

I was in a rush to send a working example to the guy on Discord asking for it and was constrained by having to pick up kids from kindergarten. So it was better working than polished :) Now I hope to have all the rough edges fixed.

File: samples/apps/websockets/application.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import uvicorn  # noqa: E402
+from websockets.sync.client import connect as ws_connect
+
+import autogen
+from autogen.io.websockets import IOWebsockets
+
+file_location = Path(__file__).parents[3] / "notebook"
+assert file_location.exists(), file_location
+
+config_list = autogen.config_list_from_json(
```
[the remark this answers is not in the document]
Comment (davorrunje) [reply samples/apps/websockets/application.py]: I used environment variables and got it working for both OpenAI and Azure deployments.

File: samples/apps/websockets/README
Comment (davorrunje) [reply samples/apps/websockets/README:1]: Oops, fixed now.

File: samples/apps/websockets/README.me
Comment (ekzhu) [inline samples/apps/websockets/README.me:1]: README.md? Not .me?

**Review by ekzhu — CHANGES_REQUESTED** [review]

File: samples/apps/websockets/README.me
Comment (davorrunje) [reply samples/apps/websockets/README.me:1]: typo 😂

**Review by ekzhu — APPROVED** [review]

**Pull request merged**